Task1/Patryk/main.cpp

```diff
@@ -0,0 +1,55 @@
+#include <iostream>
+#include <cstdio>
+
+using namespace std;
+
+bool isPrime(int value)
+{
+    bool isPrimeInd = true;
+    int i = 2;
+    do
+    {
+        if ((value % i == 0 || value < 2) && value!=2)
+        {
+            isPrimeInd = false;
+            break;
+		}
+        i++;
+    }while (i<value);
+	return isPrimeInd;
+}
+
+void primeChecker(int value)
```

Comment:
by convention function names should contain a verb. So here: checkIfPrime, printPrimeCheckResult etc.